clientv3/lease.go

```diff
@@ -58,6 +59,9 @@ type Lease interface {
 	// Grant creates a new lease.
 	Grant(ctx context.Context, ttl int64) (*LeaseGrantResponse, error)
 
+	// Lookup looks up the given lease.
+	Lookup(ctx context.Context, id LeaseID) (*LeaseLookupResponse, error)
```

Comment:
this needs a better verb since this naming will bubble up to the client-- `client.Lookup()` isn't very "leasey"